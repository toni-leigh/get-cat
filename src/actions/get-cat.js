const GET_CAT = 'GET_CAT';

const getCat = () => ({ type: GET_CAT });

export { getCat, GET_CAT };