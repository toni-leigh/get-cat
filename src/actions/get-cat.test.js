import { getCat } from './get-cat';

test('get cat returns the correct action', () => {
  expect(getCat()).toMatchSnapshot();
});