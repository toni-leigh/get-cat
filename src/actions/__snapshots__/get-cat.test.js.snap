// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get cat returns the correct action 1`] = `
Object {
  "type": "GET_CAT",
}
`;
